scripts/minishift-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+OPENSHIFT_CLIENT_BINARY_URL=${OPENSHIFT_CLIENT_BINARY_URL:-'https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz'}
```

Comment:
The client binary itself bundle within minishift binary and can be available in ```MINISHIFT_HOME``` dir once you run ```minishift start```. Use the available binary instead of downloading it to avoid more time consumption for preparing the setup.